<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1371</width>
    <height>864</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>662</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>6000</width>
    <height>6000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1241</width>
      <height>821</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1241</width>
       <height>777</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Business Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>171</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>80</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>991</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;
</string>
      </property>
      <property name="title">
       <string>                       Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>951</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Zipcode</string>
      </property>
      <widget class="QTableWidget" name="Zipcode_list">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>801</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>        Zipcode Statistics </string>
      </property>
      <widget class="QListWidget" name="Num_businesses">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string># of Businesses</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Top Categories</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="Top_cat">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>331</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>251</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>Category Filter</string>
      </property>
      <widget class="QListWidget" name="Category_list">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>221</width>
         <height>281</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Categories</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>470</y>
        <width>991</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;
</string>
      </property>
      <property name="title">
       <string>                       Popular Businesses</string>
      </property>
      <widget class="QTableWidget" name="Popular_business">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>961</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1241</width>
       <height>777</height>
      </rect>
     </property>
     <attribute name="label">
      <string>User Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 11pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>Login</string>
      </property>
      <widget class="QLineEdit" name="enter_login">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>29</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="Userid_list">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>341</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 11pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>My Information</string>
      </property>
      <widget class="QListWidget" name="display_user">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="display_stars">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="Yelp_since">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Stars:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Yelping Since:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Votes:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Funny</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Cool</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Useful</string>
       </property>
      </widget>
      <widget class="QListWidget" name="funny_votes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cool_votes">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="useful_votes">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <zorder>label_16</zorder>
      <zorder>display_user</zorder>
      <zorder>display_stars</zorder>
      <zorder>Yelp_since</zorder>
      <zorder>label_14</zorder>
      <zorder>label_15</zorder>
      <zorder>label_17</zorder>
      <zorder>label_18</zorder>
      <zorder>label_19</zorder>
      <zorder>label_20</zorder>
      <zorder>funny_votes</zorder>
      <zorder>cool_votes</zorder>
      <zorder>useful_votes</zorder>
     </widget>
     <widget class="QGroupBox" name="groupBox_16">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>731</width>
        <height>781</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 11pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>Latest Tips of My Friends</string>
      </property>
      <widget class="QTableWidget" name="latest_tips">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>711</width>
         <height>691</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>511</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 11pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>My Friends</string>
      </property>
      <widget class="QTableWidget" name="friend_table">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>471</width>
         <height>181</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>511</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 11pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>Friend Suggestions</string>
      </property>
      <widget class="QTableWidget" name="friend_suggestions">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>471</width>
         <height>251</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1371</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menua">
    <property name="title">
     <string>a</string>
    </property>
   </widget>
   <addaction name="menua"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
